### ISSUE TITLE
How is the dataset arranged?

### ISSUE THREAD

**Issue by fireWinters** [AlaaLab/InstructCV#5]
I encountered a strange issue while working with four datasets. I'd like to confirm how the four datasets are arranged?

**Comment by sunrainyg** [comment]
Hi, please refer to https://github.com/AlaaLab/InstructCV/blob/main/DATASET.md.

Feel free to ask if you have any further questions about it.

**Comment by fireWinters** [comment]
Thank you, I've resolved it. Another question: what's the purpose of the returned 'seed' in data processing?

**Comment by sunrainyg** [comment]
That is a legacy of the old version. If you look into the training code, you can find it was initially used to shuffle the training data inside each pair data. However, since in our pair data, we finally only have one original image and one ground truth image, we actually do not need that 'seed' anymore. You can ignore that :)

**Comment by fireWinters** [comment]
Great! Thanks for your detailed explanation!
